generators/app/templates/app/middlewares/errors.js

```diff
@@ -9,6 +9,11 @@ const statusCodes = {
 };
 
 exports.handle = (error, req, res, next) => {
+  if (error.name && error.name.includes('Sequelize')) {
+    logger.error(error);
+    return res.status(statusCodes[errors.DATABASE_ERROR])
+      .send({ message: error.original.message, internal_code: statusCodes[errors.DATABASE_ERROR] });
```

Comment:
`error.original.message` could contain any sensible information ? (as FK, name of columns, etc), it's a security issue return that in response body